### DISCUSSION TITLE
Incremental Clustering?

### DISCUSSION THREAD

**Discussion by drakeleeb** [alan-turing-institute/sktime#2488] [Q&A]
Are there plans to incorporate incremental (online and/or minibatch) clustering into sktime?

**Comment by fkiraly** [comment]
@chrisholder, @MatthewMiddlehurst , @TonyBagnall?

Generally, there is the `update` method across most modules, so online/stream update versions of any given task are generally in-scope for sktime.
Regarding clustering, it is relatively new and experimental, so it's probably best to ask the developers of that module on if or when.
